Islandora-CLAW/Crayfish
# patched Houdini convert method to add  when making call

**GitHub Issue**: https://github.com/Islandora/documentation/issues/2165

# What does this Pull Request do?

Tells houdini to only use the first page to generate a derivative.

# What's new?
A slightly modified imagemagick command if you're pulling images from a PDF

# How should this be tested?

Upload a big PDF (50+ MB) and wait a while.  Check the size of the thumbnail/service file that gets generated.  Apply this PR to Crayfish and do it again.  It should be faster and the image should be smaller.
